addons/website_sale/controllers/main.py

```diff
@@ -310,6 +340,10 @@ def shop(self, page=0, category=None, search='', ppg=False, **post):
             'keep': keep,
             'search_categories_ids': search_categories.ids,
             'layout_mode': layout_mode,
+            'min_price': min_price if min_price else available_min_price,
```

Comment:
`min_price` and `max_price` must be parsed to float as it might be a string if no product found (it will use the function argument which comes from the query string which is string type) and will lead to traceback.
You can try with http://localhost:8069/shop?min_price=6698.02&max_price=12429.31&search=noresultwillbefound